### PR TITLE
Added patch for yml content.

### PR DESCRIPTION
**Pull request by MdNadimHossain**
### Issue 
image

### Changes 
Added the yml content patch until the patch is approved and merged

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
   "extra": {
     "patches": {
       "drupal/yaml_content": {
-        "Alter parsed values before nodes created. - https://www.drupal.org/project/yaml_content/issues/3261689#comment-14393366": "https://www.drupal.org/files/issues/2022-02-01/yaml_content-3261689_2.patch"
+        "Alter parsed values before nodes created. - https://www.drupal.org/project/yaml_content/issues/3261689#comment-14393366": "https://www.drupal.org/files/issues/2022-02-01/yaml_content-3261689_2.patch",
+        "Warnings when importing menus using module. - https://www.drupal.org/project/yaml_content/issues/3340448": "https://git.drupalcode.org/project/yaml_content/-/merge_requests/5.diff"
```
Comment (krakerag) [inline composer.json:22]: This is probably dangerous and we should use a static diff.

https://www.drupal.org/docs/develop/git/using-git-to-contribute-to-drupal/creating-issue-forks-and-merge-requests#s-patch-files-for-use-with-composer

Comment (MdNadimHossain) [reply composer.json:22]: ah I should have used the .patch instead

Comment (krakerag) [reply composer.json]: Yeah but also, it should be a patch that isn't in a merge request.

Case in point - I could push up another commit to that MR and now your patch has changed.

Best case scenario - get the .patch, save it locally and load it from a ./patches/ folder
Second best case scenario - get the .patch, add it to the issue on D.O. and reference that URL

Comment (MdNadimHossain) [reply composer.json]: yeah I will do the 2nd one, I think will be easier to manage updates on that

**Review by anthony-malkoun — APPROVED** [review]
👍

File: composer.json [hunk 2]
```diff
@@ -18,7 +18,8 @@
   "extra": {
     "patches": {
       "drupal/yaml_content": {
-        "Alter parsed values before nodes created. - https://www.drupal.org/project/yaml_content/issues/3261689#comment-14393366": "https://www.drupal.org/files/issues/2022-02-01/yaml_content-3261689_2.patch"
+        "Alter parsed values before nodes created. - https://www.drupal.org/project/yaml_content/issues/3261689#comment-14393366": "https://www.drupal.org/files/issues/2022-02-01/yaml_content-3261689_2.patch",
+        "Warnings when importing menus using module. - https://www.drupal.org/project/yaml_content/issues/3340448": "https://www.drupal.org/files/issues/2023-02-14/warnings-when-importing_3340448_4.patch"
```
Comment (krakerag) [inline composer.json:22]: Super, thanks @MdNadimHossain. Sorry to be a pest with small things like security :)

**Review by krakerag — APPROVED** [review]
LGTM

File: composer.json (hunk 2 above)
Comment (MdNadimHossain) [reply composer.json:22]: all good, all these suggestions are valid :D

**Pull request merged**